x-pack/examples/ui_actions_enhanced_examples/public/dashboard_to_url_drilldown/index.tsx

```diff
@@ -101,7 +102,15 @@ export class DashboardToUrlDrilldown implements Drilldown<Config, ActionContext>
     return config.url;
   };
 
-  public readonly execute = async (config: Config, context: ActionContext) => {
+  public readonly execute = async (
+    config: Config,
+    context: ActionExecutionContext<ActionContext>
```

Comment:
If we want tot use `trigger`, we "specify" special type here which is an extension around regular `ActionContext`